src/common/ImageAvatar.js

```diff
@@ -32,7 +41,9 @@ export default class ImageAvatar extends PureComponent {
 
     return (
       <Touchable onPress={onPress} style={touchableStyle}>
-        <Image style={imageStyle} source={{ uri: avatarUrl }} resizeMode="contain" />
+        <Image style={imageStyle} source={{ uri: avatarUrl }} resizeMode="contain">
+          {status && <UserStatusIndicator style={componentStyles.status} status={status} />}
```

Comment:
Actually, I think we want to keep this in the Avatar.js component. Any reason to prefer moving it here?